packages/docs/stories/templates/form.stories.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import type { Meta } from '@storybook/web-components';
+import { Title, Subtitle, Description, Stories } from '@storybook/blocks';
+import { html } from 'lit';
+
+const meta: Meta = {
+  parameters: {
+    docs: {
+      page: () => (
+        <>
+          <Title />
+          <Subtitle />
+          <Description />
+          <Stories />
+        </>
+      ),
+    },
+  },
+  title: 'Templates/Forms',
+};
+export default meta;
+
+export const ContactForm = {
+  render: () => html`
+    <style>
+    .synergy-form-demo {
+      background: var(--syn-color-neutral-0);
+      border: 1px dashed var(--syn-color-neutral-300);
+      margin: 0 auto;
+      padding: 10px 20px;
+      max-width: 750px;
+      min-width: 360px;
+    }
+
+    .synergy-form-demo h1 {
+      font-size: var(--syn-font-size-3x-large);
+      font-weight: normal;
+      margin: 0 0 10px 0;
+    }
+
+    .synergy-form-demo > p {
+      margin: 0 0 var(--syn-spacing-2x-large) 0;
+    }
+
+    .synergy-form-demo em {
+      font-size: var(--syn-font-size-x-small);
+      font-style: normal;
+    }
+
+    .synergy-form-demo fieldset {
+      border: none;
+      margin: 0 0 var(--syn-spacing-2x-large) 0;
+      padding: 0;
+    }
+
+    .synergy-form-demo fieldset legend {
+      font-size: var(--syn-font-size-large);
+      margin-bottom: var(--syn-font-size-small);
+    }
+
+    .synergy-form-demo fieldset legend + p {
+      font-size: var(--syn-font-size-medium);
+      margin: 0 0 var(--syn-font-size-medium) 0;
+    }
+
+    .synergy-form-demo .fields {
+      display: flex;
+      flex-flow: wrap;
+      margin-bottom: var(--syn-font-size-x-large);
+      gap: var(--syn-spacing-large) var(--syn-font-size-small);
+    }
+
+    .synergy-form-demo .fields > * {
+      flex-basis: 100%;
+    }
+
+    @media screen and (min-width: 641px) {
+      .synergy-form-demo .fields > * {
+        flex-basis: calc(50% - var(--syn-spacing-medium));
+      }
+    }
+
+    /*
+     * The form control inputs are using a css grid to be displayed.
+     * We just let them flow automatically mobile and arrange them
+     * in two columns when space is wide enough
+     */
+    syn-radio-group::part(form-control-input) {
+      display: grid;
+    }
+
+    @media screen and (min-width: 641px) {
```

Comment:
As one principle of us is `Progressive` I would suggest to go the Container Query Route, instead of looking on screen sizes – this makes especially sense for multi column layouts. Support ist quite fine and I think we should lead by example for Progressive Enhancement: https://caniuse.com/css-container-queries

In fact that's something where we have to align with Design on.

Reply:
I will try it out and adapt the story.